app/ai/voice/agents/breeze_buddy/utils/common.py

```
@@ -404,87 +404,183 @@ def get_validation_error_message(errors: List[str]) -> str:
+                x_new = np.linspace(0, len(data) - 1, new_length)
+                data = np.interp(x_new, x_old, data)
+
+            # Convert to int16 (same format SoundfileMixer expects internally)
+            int16_data = np.clip(data * 32767, -32768, 32767).astype(np.int16)
+            self._sounds[sound_name] = int16_data
+
+            logger.info(
+                f"[ResamplingMixer] Loaded '{sound_name}': "
+                f"{len(int16_data) / self._sample_rate:.1f}s at {self._sample_rate}Hz"
+            )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)